<?xml version="1.0"?>
<launch>

    <arg name="joystick_config" default="f710" doc="possible values: f710, ps4"/>

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="autorepeat_rate" value="10"/>
        <param name="coalesce_interval" value="0.10"/>
    </node>


    <node type="tank_teleop.py" pkg="ocg120g_teleop" name="tank_teleop" output="screen">
        <rosparam command="load" file="$(find ocg120g_teleop)/config/$(arg joystick_config).yaml"/>
    </node>

</launch>